app/src/main/java/com/android/voyageur/ui/formFields/UserDropdown.kt

```diff
@@ -99,9 +99,10 @@ fun UserDropdown(
             DropdownMenuItem(
                 text = { Text("${userPair.first.name} ") },
                 onClick = { onUpdate(userPair, index) },
-                leadingIcon = {
-                  Checkbox(
-                      checked = userPair.second, onCheckedChange = { onUpdate(userPair, index) })
+                trailingIcon = {
+                  Button(onClick = {}) { Text("Send trip request") }
+                  // Checkbox(
+                  ///    checked = userPair.second, onCheckedChange = { onUpdate(userPair, index) })
```

Comment:
why is this change needed here?

Reply:
It's not, thanks for noticing! This was for the other feature I am working on, I think I forgot to change branches... I'll revert this..